css-three-column-layout/styles.css

```diff
@@ -0,0 +1,84 @@
+.row {
+  display: flex;
+  flex-basis: auto;
+  justify-content: center;
+  background-color: #1996da;
+  padding-bottom: 25px;
+}
+
+h2 {
+  display: block;
+  text-align: center;
+  justify-content: center;
+  opacity: 0.9;
+}
+
+.white-text {
+  color: white;
+}
+
+img {
+  background-color: white;
+  width: 50%;
+}
+
+.font {
+  font-family: Lato, sans-serif;
+}
+
+p {
+  margin: auto;
+  padding: 0 10px 10px;
+  color: #d0e8f7;
+  font-size: 75%;
+  text-align: center;
+}
+
+h4 {
+  text-align: center;
+  color: #daeef9;
+}
+
+.marketing-container {
+  display: flex;
+}
+
+.col-third {
+  display: flex;
+}
```

Comment:
`column` classes should only have `flex-basis` applied to them. Why does this column need to be a flexbox? All of its content is vertical, you can make use of normal `block` and `inline` document flow